tests/e2e/tests/protoMsgToJSON.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2022-2025, Chain4Travel AG. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package tests
+
+import (
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+)
+
+/*
```

Comment:
golang best practice usually suggest to use block comments only for big file-level comments on the top (so called doc comments) and the rest of the comments to do with line comments. I'd suggest to follow that to keep consistent style of function-comments across project
(this PR comment is also valid for all other similar comments in code)

Reply:
Alright, changed all the multi-line to single line comments.